src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const newVariable = person.sex === 'm' ? (
```

Comment:
It is not a mistake but in the future try to use more descriptive vars names. for instance, here it could be `partnersInfo` or something. It is a good practice